Analytics/Classes/Integrations/SEGIntegrationsManager.m

```diff
@@ -70,7 +70,7 @@ - (instancetype _Nonnull)initWithAnalytics:(SEGAnalytics *_Nonnull)analytics
         self.configuration = configuration;
         self.serialQueue = seg_dispatch_queue_create_specific("io.segment.analytics", DISPATCH_QUEUE_SERIAL);
         self.messageQueue = [[NSMutableArray alloc] init];
-        self.httpClient = [[SEGHTTPClient alloc] initWithRequestFactory:configuration.requestFactory];
+        self.httpClient = [[SEGHTTPClient alloc] initWithRequestFactory:configuration.requestFactory configFactory:configuration.configFactory];
```

Comment:
Seems like we need a better name for `configFactory`. may be `sessionConfigFactory`?

Reply:
Sure.